### PR TITLE
Fixes for SCRIPTDOC-19, SCRIPTDOC-23, SCRIPTDOC-25, SCRIPTDOC-26, SCRIPTDOC-27

### PR DESCRIPTION
**Pull request by raphj**

### CODE REVIEW

**Comment by lucaa** [comment]
@raphj so this is not ready to merge yet?

What is the regression you mention above? Is it visual only? If so, I guess we can live with it and merge this PR if this is the only thing blocking it.

**Comment by Sereza5** [comment]
https://github.com/user-attachments/assets/accc02ac-34f5-4393-9a1d-d6cc5b128924

With the information shared, I tried to hit the regression but could not find anything relevant (see video above).
In any case, I doubt this would be a major regression and IMO it's worth to fix the large regression on style that is SCRIPTDOC-25.

___
After reviewing the code, it looks like this scrolling could be handled natively in modern browsers...
The CSS could probably be a bit shorter/better organized, but it's also probably not worth spending a lot of time on it.
500+ lines of CSS for one page looks a bit excessive.
____
TLDR. It does not look like very maintainable code but it works and positives far outweight negatives so I'm in favor of merging it.

**Comment by raphj** [comment]
@Sereza7 thanks for trying to reproduce the regression and for the code review. I'm looking into it this morning, will try to sort things out and merge.

**Comment by raphj** [comment]
@Sereza7 I think I fixed everything. Would you mind double checking if things work on your side before I merge?

**Comment by Sereza5** [comment]
I copy pasted the changes on the version I have installed on my instance (not too long to do since it's just a couple files).

The only difference I pointed out is that there are no scrolling animations anymore, as expected according to https://github.com/xwiki-contrib/application-scripting-documentation/pull/6/commits/c49296ec4dd840ee11b10ae20dd0bc925d86706b 's description :+1: 
Elsewise, things looked alright when I tried to use it. I didn't take the time to manual test every ticket but it's definitely an improvement.

**Comment by raphj** [comment]
thanks @Sereza7 for the quick test!
